src/Security/Authentication/JwtBearer/src/JwtBearerConfigureOptions.cs

```diff
@@ -84,10 +84,14 @@ private static IEnumerable<SecurityKey> GetIssuerSigningKeys(IConfiguration conf
     {
         foreach (var issuer in issuers)
         {
-            var keyFromSecret = configuration[$"{issuer}:KeyMaterial"];
-            if (!string.IsNullOrEmpty(keyFromSecret))
+            var signingKeys = configuration.GetSection("SigningKeys")
+                .GetChildren()
+                .Where(key => key["Issuer"] == issuer)
+                .Select(key => key["Value"])
+                .OfType<string>();
+            foreach (var signingKey in signingKeys)
```

Comment:
Are multiple keys per issuer allowed?

Reply:
It's possible for a single issuer/authorization server to produce a set of signing keys that the application can use with no preference between them ([ref](https://datatracker.ietf.org/doc/html/rfc7517#section-5.1)). Most of what I've seen about this touches on them being used for OIDC scenarios, but it helps to support it in some way with JWT since we want the local dev scenario to map nicely to the pod scenario.